packages/editor/src/store/selectors.js

```diff
@@ -1721,3 +1735,18 @@ export function __experimentalGetTemplateInfo( state, template ) {
 		icon: templateIcon,
 	};
 }
+
+/**
+ * Helper function to find the corresponding icon for a template part's 'area'.
+ *
+ * @param {string} area The value of the template part 'area' tax term.
+ *
+ * @return {Object} The corresponding icon.
+ */
+function getTemplatePartIconByArea( area ) {
```

Comment:
This is not a selector, but a `util`. Can we move this to another file?

Reply:
yup, updated!